### ISSUE TITLE
Your attached binary asset to the release 1.0.8 is a .tar with a .tar.gz extension

### ISSUE THREAD

**Issue by maxhbr** [microsoft/component-detection#40]
Just a minor thing but maybe surprising:

I looked at https://github.com/microsoft/component-detection/releases/download/v1.0.8/component-detection.tar.gz with
```
$ md5sum component-detection.tar
21ff593d6c89fb1f55f6035526c52a82  component-detection.tar
```
and it is just a tar:
```
$ file component-detection.tar.gz
component-detection.tar.gz: POSIX tar archive (GNU)
```

**Comment by JamieMagee** [comment]
Whoops! Thanks for reporting. Looks like it's happening here:

https://github.com/microsoft/component-detection/blob/8b392cef89745f06b69c515838cb6fd04a2c0132/.github/workflows/build-release.yml#L40-L60

I'm probably going to drop this artifact from releases going forward (See #37). It contains platform-independent, runtime dependent binaries i.e. runs on any OS that has a .NET Core 3.1 runtime installed. Instead I'll be uploading platform-dependent, runtime-independent binaries i.e. OS specific, but doesn't require a .NET Core 3.1 rune to be installed.

I think this will be easier to get up and running with, but happy to hear what would work best for you?

**Comment by maxhbr** [comment]
That sounds good, the platform-dependent binaries are probably better

**Comment by JamieMagee** [comment]
Closed by #37
